### PR TITLE
Removes printing photos from security consoles

### PR DESCRIPTION
**Pull request by Birdtalon**
I was asked to remove this by @Spartan6 apparently it causes people to crash en-masse when used.

🆑 
rscdel: Removes printing photos on security consoles.
/🆑

### CODE REVIEW

**Comment by Spartan6** [comment]
Finally the chases that pause midway through happening due to the antag crashing, then the security running up and crashing, then the antag reconnecting and then the security reconnecting can cease.

**Comment by Aurorablade** [comment]
No. Again the problem is not the photo but something being passed in the live data to the photo to generate it. I cannot reproduce this on a local test or the paradise test server. So the issue lies somewhere in the data the photo gets on a server with alot of people. Question is where

File: code/game/machinery/computer/security.dm [hunk 1]
```diff
@@ -369,15 +369,6 @@
 				P.name = "paper - 'Security Record'"
 				printing = 0
 
-		else if(href_list["print_p"])
```
Comment (tigercat2000) [inline code/game/machinery/computer/security.dm]: You should probably just comment this out, make a BYOND bug report, and put in a comment `//disabled due to byond bug <link here>`

File: code/game/machinery/computer/security.dm [hunk 2]
```diff
@@ -369,15 +369,6 @@
 				P.name = "paper - 'Security Record'"
 				printing = 0
 
-		else if(href_list["print_p"])
-			if(!printing)
-				printing = 1
-				playsound(loc, "sound/goonstation/machines/printer_dotmatrix.ogg", 50, 1)
-				sleep(50)
-				if(istype(active1, /datum/data/record) && data_core.general.Find(active1))
-					create_record_photo(active1)
```
Comment (tigercat2000) [inline code/game/machinery/computer/security.dm]: Also get rid of or comment out this proc, wherever it is

**Comment by Spartan6** [comment]
I'm sure once you find the issue you cannot find Fethas, you'll be able to add them back in. For the moment, having something that can crash dozens of users at once in game is not a good thing.

**Comment by Anasari** [comment]
It's about time this get removed, even if temporarily.

**Comment by StreakyHaddock** [comment]
I know it's not the photo that's the damn issue, Fethas, but it's a huge headache to deal with. It's a feature that's not used all the time and few people will miss it, as it's not crucial. It's definitely a useful function, but it's absolutely not worth it considering the bloody headaches it creates. Please, please disable this at least until the cause is found.

**Comment by Birdtalon** [comment]
I've commented out the parts for the security console relating to printing photos. This proc is used nowhere else, however I've commented out the code anyway.

I don't really have time to do any real debugging to try and find the issue behind this, so maybe @Aurorablade can make a BYOND bug report as she knows a bit more about it than I?

**Comment by Vivalas** [comment]
Ooh.. how long ago was the correlation between this and crashes discovered? And what kind of crash is this, the one that just hits you randomly sometimes and freezes your screen but nothing else?

If this is indeed related to *that* crash bug than Lummox should perhaps be informed. Well he should be anyways if this is indeed entirely a BYOND bug.

**Comment by Spartan6** [comment]
Feel free to write up the Byond bug, but it’s been a problem for a while.

**Pull request merged**